Configurations/Parse-iOS-Dynamic.xcconfig

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright (c) 2015-present, Parse, LLC.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+//
+
+PRODUCT_NAME                    = Parse
+IPHONEOS_DEPLOYMENT_TARGET      = 8.0
+// This silence a warning
+//MACOS_DEPLOYMENT_TARGET           = 8.0
+SDKROOT[sdk=macos*]             = iphoneos
+SDKROOT[sdk=iphoneos*]          = iphoneos
+SDKROOT[sdk=iphonesimulator*]   = iphonesimulator
+MACH_O_TYPE                     = mh_dylib
+
+ENABLE_BITCODE = NO
```

Comment:
I am somewhat sceptical about bitcode here actually.
Can you verify that the final binary actually has bitcode?